dist/linux/systemd.service.in

```diff
@@ -1,9 +1,12 @@
 [Unit]
 Description=@NAME@
 After=graphical-session.target
+After=dbus.socket
+Requires=dbus.socket
 
 [Service]
-Type=dbus
+Type=notify-reload
+ReloadSignal=SIGUSR2
 BusName=@APPID@
```

Comment:
This is what tells `systemd` that it should send the `SIGUSR2` signal to the main PID when a reload is requested. Normally it sends `SIGHUP`.